### DISCUSSION TITLE
does not allow uploading webp images for articles

### DISCUSSION THREAD

**Discussion by AlonsoMartinezh** [openstore-ecommerce/OpenStore#173] [General]
the extension is enabled in dnn and the module but it does not allow to load this type of file

**Comment by leedavi** [comment]
Sorry about the late reply.

There is a setting in "BackOffice>Admin>Setting"  That allows you to add regexpr for any type you need.

![image](https://user-images.githubusercontent.com/5583141/128148411-641701d7-4b8f-4699-a947-0550427765c3.png)

**Reply by leedavi** [reply]
Although I'm unsure if the resize on load will work.

I'm also unsure about how the thumbnails will display those.  You may have to change the templates to looks at the images directly.

**Comment by AlonsoMartinezh** [comment]
Thanks for the information, just as in DNN make this configuration but it does not work and does not generate an error .
![file_webp](https://user-images.githubusercontent.com/13344249/128283434-0e8846a8-49c8-4604-bf4b-2e76e2ed79a6.png)

**Reply by DNNMonster** [reply]
I think the biggest hurdle is that the current ResizeImage function doesn't handle the webp format.

If you mitigate the js and server side validation errors you'll still end up bumping into issues with instantiating a Bitmap from the webp format within NBrightCore's ImageUtilities ResizeImage func.   System.Drawing was born before webp was a sparkle in Google's eyes. It doesn't know how to deal with the rascal.

I believe it would require introducing dependencies to libwebp and something like Imazen.WebP package to enable support.  

It's quite possible I'm overlooking a better path..??
